docs/en/observability/apm/getting-started-apm/index.asciidoc

```diff
@@ -49,12 +40,11 @@ integrations from one central {fleet} UI.
 a| * {es}
 * {ess}
 
+NOTE: Fleet-managed APM Server does support all the outputs that are supported by the APM Server binary method of running Elastic APM.
```

Comment:
This is not true - Fleet managed APM Server does **only** support Elasticsearch output. 

>The missing support for outputs different than Elasticsearch is the only thing that comes to mind ad-hoc, but I also haven't spent much time on this recently. We should still be calling out that we only support ES as output when running under Fleet.

Reply:
```suggestion
NOTE: Fleet-managed APM Server does _not_ support all the outputs that are supported by the APM Server binary method of running Elastic APM.
```